src/main/java/org/broadinstitute/consent/http/service/EmailService.java

```diff
@@ -143,12 +144,14 @@ protected void sendMessage(MailMessage mailMessage, Integer userId) throws IOExc
         content);
   }
 
-  public List<org.broadinstitute.consent.http.models.mail.MailMessage> fetchEmailMessagesByType(EmailType emailType, Integer limit,
+  public List<org.broadinstitute.consent.http.models.mail.MailMessage> fetchEmailMessagesByType(
+      EmailType emailType, Integer limit,
       Integer offset) {
     return emailDAO.fetchMessagesByType(emailType.getTypeInt(), limit, offset);
   }
```

Comment:
If you want to include format-only changes, can you reformat using the google-java-format plugin (or by running spotless locally) so the resulting code conforms to the google java format?
```suggestion
  public List<org.broadinstitute.consent.http.models.mail.MailMessage> fetchEmailMessagesByType(
      EmailType emailType, Integer limit, Integer offset) {
    return emailDAO.fetchMessagesByType(emailType.getTypeInt(), limit, offset);
  }
```